test/python/test_roles_api.py

```diff
@@ -727,5 +728,41 @@ def test_delete_member_422(self):
 
         self.assertEqual(context.exception.status, 422)
 
+    def test_get_role_graph_200(self):
```

Comment:
In these tests, the cases for each function are marked by a comment for readability.

```suggestion
    # Test cases for getting a role's member/membership graph
    
    def test_get_role_graph_200(self):
```

Reply:
I don't see what this type of separation of comment actually adds to the file. A user would know what functionality the tests are assessing based off of the name of the test and the response code included in the method and comment. Structural comments like these more often than not end up getting ignored and can very easily turn into lying comments down the road. I agree that the tests need to be separated into their own `TestCase`'s. I just opened an issue a few days ago to separate things, but that is out of scope for this PR.

Reply:
I dig separate `TestCase`'s for each. Great idea.